cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/AbstractStorageProviderNamespaceAdmin.java

```diff
@@ -69,9 +69,7 @@ public void create(NamespaceMeta namespaceMeta) throws IOException, ExploreExcep
 
     createLocation(namespaceMeta);
 
-    // only create non-default namespaces in Hive
-    if (cConf.getBoolean(Constants.Explore.EXPLORE_ENABLED) &&
-      !NamespaceId.DEFAULT.equals(namespaceMeta.getNamespaceId())) {
```

Comment:
Has this been tested on a cluster. What's the behavior if the database already exists.